File: problems/3110. Score of a String/c++/main.cpp
#include <cmath>
#include <iostream>
#include <string>

using namespace std;
class Solution {
public:
  int scoreOfString(string s) {
    int score = 0;

    for (int i = 0; i < s.size() - 1; i++) {
      score += abs(s[i] - s[i + 1]);
    }
    cout << "score: " << score << "\n";
    return score;
  }
};

int main() {
  auto s = Solution();
  s.scoreOfString("hello");
}

File: problems/3110. Score of a String/python3/main.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        score = 0

        for i in range(len(s) - 1):
            score += abs(ord(s[i]) - ord(s[i + 1]))

        return score


if __name__ == "__main__":
    s = Solution().scoreOfString("hello")
    print(s)
    assert s == 13
